packages/website/src/components/OptionsSelector.tsx

```diff
@@ -24,6 +25,8 @@ export interface OptionsSelectorParams {
   readonly tsVersions: readonly string[];
 }
 
+const MIN_TS_VERSION_SEMVER = '>=4.7.5';
```

Comment:
[Bug] 4.7 isn't being given as an option in the playground, was this meant to be 4.7.4?

![Playground TypeScript version selector showing minors from 4.8 up through 5.3](https://github.com/typescript-eslint/typescript-eslint/assets/3335181/f8d7e213-f91d-45ec-a33d-da36c35a53b1)

```suggestion
const MIN_TS_VERSION_SEMVER = '>=4.7.4';
```